<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gerenciaDispositivo</class>
 <widget class="QDialog" name="gerenciaDispositivo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>162</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>362</width>
    <height>162</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>362</width>
    <height>162</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Buscar Ativos</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineID"/>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="toolButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Find.ico</normaloff>Find.ico</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineNome"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Tipo</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboTipo">
      <property name="maxVisibleItems">
       <number>5</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAlphabetically</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Local</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineLocal"/>
    </item>
    <item row="4" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>showEditDispositivos()</slot>
 </slots>
</ui>
